internal/bminventory/inventory.go

```diff
@@ -110,7 +110,7 @@ type Config struct {
 	DiskEncryptionSupport           bool              `envconfig:"DISK_ENCRYPTION_SUPPORT" default:"true"`
 	// TODO: remove when baremetal will be supported in arm
 	// this env allows to set specific image to extract openshift-baremetal-install
-	InstallerReleaseImageOverrideUnsupported string `envconfig:"INSTALLER_RELEASE_IMAGE_OVERRIDE_UNSUPPORTED" default:""`
+	AllowInstallerReleaseImageOverride bool `envconfig:"ALLOW_INSTALLER_RELEASE_IMAGE_OVERRIDE" default:"false"`
```

Comment:
The comment above is off.
Also, why do we need this env anyway, isn't this PR adding the support?

Reply:
sort of, we want to be able to disable support till we don't have multi tenant feature flags